src/search.cpp

```diff
@@ -179,22 +179,21 @@ namespace {
 
 void Search::init() {
 
-  const double K[][2] = {{ 0.799, 2.281 }, { 0.484, 3.023 }};
```

Comment:
Please use code self-documenting constant:

    const bool PV = true;

and use PV and !PV instead of 0, 1 like:

    if (!imp && Reductions[!PV][imp][d][mc] >= 2 * ONE_PLY)
               Reductions[!PV][imp][d][mc] += ONE_PLY;